fesvr/htif.cc

```diff
@@ -157,7 +162,11 @@ void htif_t::load_program()
   for (auto payload : payloads)
   {
     reg_t dummy_entry;
-    load_payload(payload, &dummy_entry);
+    load_payload(payload, &dummy_entry, is32bit);
+    if(is32bit != (32 == max_xlen)){
+      printf("loading %dbit payload on %dbit core!\n", is32bit?32:64, max_xlen);
```

Comment:
“Error: cannot execute %d-bit program on RV%d hart”

Reply:
Changed to using this format.